src/Shared/UnitTests/FileUtilities_Tests.cs

```diff
@@ -724,7 +724,7 @@ public void GenerateTempFileNameWithExtension()
 
             try
             {
-                path = Shared.FileUtilities.GetTemporaryFile(".bat");
+                path = FileUtilities.GetTemporaryFile(".bat");
```

Comment:
So the idea is that with more changes like this, we can remove a bunch of the shared source files from compiling in every test project?